meetings/2025-03-19.md

```diff
@@ -0,0 +1,109 @@
+
+
+### Scribes
+* Michiel
+
+## Topics
+
+### Moving AddressBook from vcard to foaf
+
+* MdJ: See https://github.com/solid/contacts/issues/8Md
+* MdJ: ...
+* MdJ: VCard is controlled by IETF. If we change W3C ontology it will result in a fork.
+  ... https://lists.w3.org/Archives/Public/semantic-web/2025Mar/0041.html
+  ... This topic pops up in discussions since 1999
+* JW: I'm looking at VCard as RFC which was published in 2011. We could extend vocabulary which is a common practice. We can have `solid-contacts` namespace and define additional terms there.
+* MdJ: This is close to Dan's proposal of moving AddressBook to FOAF. Address books are not specific to Solid. FOAF has similar terms to VCard. 
+* JW: FOAF already has alternatives to Vcard...
+  ... I see Tim's disagreement to use FOAF
+* MdJ: Instead of migrating it to FOAF we
+* eP: What is a problem with reusing SIOC AddressBook?
+* MdJ: We have code that uses a term. What went wrong is squating vcard ontolgy. The mechanism that address book uses is different from what SIOC is using. We just want to publis ontology which is used by existing implementations.
+* ... databases have migrations, we shouldn't instead of just rename it have to use a different mechanism.
+* eP: There is no existing stable shape for address books. One implementation can't expect that everyone else will adopt their choice. We need a curation process which we partially discussed in shaperepos meeting. 
+* HZ: Why do we need to migrate? We should allow on the fly conversions in the infrastructure. This will allow using new standards that may be developed in the future.
+* MdJ: What makes it special is that terms used were never published and can't be published for political reason. If we could publish we wouldn't need to migrate. The main problem is the squating.
+* eP:...
+* JW: We have been discussin ongology alignment and using it in Solid for years. We don't have infrastructure for that so we need an alignment to have interop.
+* ...: As for getting everyone together, we need more lightweight way of finding consensus. If we need a series of calls this will not neccessarly lead to consensus. I'm thinking about more lightweight alternative. I see 2 or 3 options. We can create new ontoloty that extends Vcard. Another is adding it to FOAF and migrating to FOAF. The last one would be creating existing contacts ontology. We should make a straw poll to see if there is interest in using one of those approaches. Hearing Tim's comments I would prefer keeping VCard and extending it.
+* MdJ: see https://lists.w3.org/Archives/Public/semantic-web/2025Mar/0031.html
+* ...: If we don't change the name and just change the prefix it would be easy to change.
+* JW: My suggestion is not listed in your email. I propose keepting vcard and just creating `vcard-extended` where we can define `AddressBook`.
+* MdJ: Are you saying that this is non-breaking change, so we are going to use `http://www.w3.org/2006/vcard/ns#` namespace?
```

Comment:
```suggestion
* ...: If we don't change the name and just change the prefix, it would be easy to change.
* JW: My suggestion is not listed in your email. I propose keeping Vcard and just creating `vcard-extended` where we can define `AddressBook`.
* MdJ: Are you saying that this is non-breaking change, so we are going to use `http://www.w3.org/2006/vcard/ns#` namespace?
```